cinema/tests/test_movie_api.py

```diff
@@ -155,5 +155,5 @@ def test_image_url_is_shown_on_movie_session_detail(self):
             ntf.seek(0)
             self.client.post(url, {"image": ntf}, format="multipart")
         res = self.client.get(MOVIE_SESSION_URL)
-
+        print(f"\n {res.data} \n")
```

Comment:
There is a debug `print` statement here. It's good practice to remove such statements before finalizing or deploying your tests to keep the output clean.